public/js/controller/pages/about/about.ctrl.js

```diff
@@ -1,7 +1,8 @@
 auroraApp.controller('AboutCtrl', function($scope, $controller) {
   angular.extend(this, $controller('LoadableMixin', {$scope: $scope}));
```

Comment:
There's no need to extend LoadableMixin here.  We don't need to get anything asynchronously.  This was just there as an example.